client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -1,2 +1,27 @@
 {
+  "graphQLTransformer": {
+    "description": "Feature Flag related to GraphQL Transformer",
+    "features": {
+      "addMissingOwnerFields": {
+        "description": "Automatically add owner field to when owner fields are not in the type",
```

Comment:
the 'to' seems to be an extra, without it: "Automatically add owner field when owner fields are not in the type"